alerting/src/main/kotlin/org/opensearch/alerting/DocumentReturningMonitorRunner.kt

```diff
@@ -81,19 +84,26 @@ object DocumentReturningMonitorRunner : MonitorRunner {
         for (i: Int in 0 until count) {
             val shard = i.toString()
             val maxSeqNo: Long = getMaxSeqNo(monitorCtx.client!!, index, shard)
-            updatedLastRunContext[shard] = maxSeqNo.toString()
+            updatedLastRunContext[shard] = maxSeqNo
+
+            // update lastRunContext if its a temp monitor as we only want to view the last bit of data then
+            if (isTempMonitor) {
+                lastRunContext[shard] = max(-1, maxSeqNo - 1)
+            }
```

Comment:
For dry run cases we would instead want to always pick latest data, such as the last 100 documents from the ending seq no. Thoughts?
Not a blocker for this PR, as we can add a ToDo here for now, and can build this separately.

Reply:
Sounds good, I will add this as a TODO and the merge it